dask_ml/model_selection/methods.py

```diff
@@ -162,13 +169,23 @@ def cv_n_samples(cvs):
     return cvs.num_test_samples()
 
 
-def cv_extract(cvs, X, y, is_X, is_train, n):
-    return cvs.extract(X, y, n, is_X, is_train)
+def cv_extract(cvs, X, y, is_X, is_train_folds, n):
+    return cvs.extract(X, y, n, is_X, is_train_folds)
+
 
+def cv_extract_params(cvs, keys, vals, n, is_train_folds):
+    '''
+        cv_extract_params the fit parameters of the fold sets (train folds or test fold)
 
-def cv_extract_params(cvs, keys, vals, n):
-    return {k: cvs.extract_param(tok, v, n) for (k, tok), v in zip(keys, vals)}
+        cvs: (CVCache): CV cache for CV information of folds
+        keys: ((str,str)) fit params (name,full_name) key tuple
```

Comment:
I think having a key that is just the `full_name` or `name` of the fit parameter key is simpler but `v1.3.0` of dask-ml uses  this tuple as the key and I am avoiding changing this key format as I don't want to find out want to find out what the downstream impact is on this change for the sake of a simpler key scheme. This is also why `_generate_fit_params_key_vals` was necessary